EKF/ekf_helper.cpp

```diff
@@ -1623,6 +1624,7 @@ void Ekf::calcExtVisRotMat()
 	// convert filtered vector to a quaternion and then to a rotation matrix
 	q_error.from_axis_angle(_ev_rot_vec_filt);
 	q_error.normalize();
+	q_error.canonicalize();
```

Comment:
This step appears unnecessary. If a canonical form is required for an external accessor, then the operation should be done there.